3567/0/about_dice_project.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+# Implement a DiceSet Class here:
+#
+# class DiceSet
+#   code ...
+# end
+class DiceSet
+  attr_reader :values
+  def roll(num)
+    @values = []
+    (1..num).each { |_count| @values << rand(1..6) }
+  end
+  @values
```

Comment:
remove this line. You have an instance variable, it's not necessary to return something here.